libeventfilter/EventLogFilterManager.cpp

```diff
@@ -0,0 +1,159 @@
+
+#include "EventLogFilterManager.h"
+#include "Common.h"
+#include <libblockchain/BlockChainInterface.h>
+#include <libethcore/CommonJS.h>
+#include <libeventfilter/EventLogFilterParams.h>
+
+using namespace std;
+using namespace dev;
+using namespace dev::event;
+using namespace dev::eth;
+
+// start EventLogFilterManager
+void EventLogFilterManager::start()
+{
+    if (m_isInit)
+    {  // already start
+        return;
+    }
+    startWorking();
+    m_isInit = true;
+    EVENT_LOG(INFO) << LOG_BADGE("EventLogFilterManager") << LOG_DESC("start");
+}
+
+// stop EventLogFilterManager
+void EventLogFilterManager::stop()
+{
+    if (m_isInit)
+    {
+        stopWorking();
+        m_isInit = false;
+        EVENT_LOG(INFO) << LOG_BADGE("EventLogFilterManager") << LOG_DESC("stop");
+    }
+}
+
+// main loop thread
+void EventLogFilterManager::doWork()
+{
+    // add new EventLogFilter to m_filters
+    addFilter();
+    executeFilters();
+}
+
+EventLogFilterManager::filter_status EventLogFilterManager::executeFilter(
+    EventLogFilter::Ptr _filter)
+{
+    EVENT_LOG(DEBUG) << LOG_KV("groupID", _filter->getParams()->getGroupID());
+    // ADD TO DO
```

Comment:
```suggestion
    // TODO: to do what
```
this type of TODO will be highlight😄

Reply:
to do next pr 😄~